src/Utils/Jalasoft.TeamUp.Projects.ProjectsException/ErrorMessage.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Jalasoft.TeamUp.Projects.ProjectsException
+{
+    using System.Collections.Generic;
+    using FluentValidation;
+
+    public class ErrorMessage
+    {
+        public IEnumerable<FluentValidation.Results.ValidationFailure> Errors { get; set; }
```

Comment:
we should return a custom TeamUp object that has the following only:
{
    "errors": [
        {
            "propertyName": "Name",
            "errorMessage": "The length of 'Name' must be 2 characters or fewer. You entered 6 characters.",
            "attemptedValue": "TeamUp"
        },
        {
            "propertyName": "Description",
            "errorMessage": "'Description' must not be empty.",
            "attemptedValue": null
        }
    ],
    "message": "Please review the errors, inconsistent data."
}

Reply:
done